lightning/src/ln/channelmanager.rs

```diff
@@ -10503,6 +10503,40 @@ where
 			"Dual-funded channels not supported".to_owned(),
 			 msg.channel_id.clone())), *counterparty_node_id);
 	}
+
+	fn handle_message_received(&self) {
+		let invoice_requests = self.pending_outbound_payments.release_invoice_request_awaiting_invoice();
+		if invoice_requests.is_empty() {
+			return;
+		}
+
+		let invoice_requests_reply_paths = invoice_requests
+			.into_iter()
+			.filter_map(|(payment_id, invoice_request)| {
+				let context = OffersContext::OutboundPayment { payment_id: payment_id };
```

Comment:
```rust
let context = OffersContext::OutboundPayment { payment_id };
```

Reply:
Addressed in [pr3010.18](https://github.com/shaavan/rust-lightning/commits/pr3010.18)
Thanks!